templates/zerver/config_error.html

```diff
@@ -97,6 +97,7 @@ <h1 class="lead">Configuration error</h1>
                     {% endif %}
                     <p>After making your changes, remember to restart
                     the Zulip server.</p>
+                    <p><a href=""> Refresh</a> to try again or <a href="/login">click here</a> to go back to login page.</p>
```

Comment:
I'm not sure if this sounds good. We might want to change it to have better wording.

Reply:
```suggestion
                    <p><a href=""> Refresh</a> to try again or <a href="/login/">click here</a> to go back to the login page.</p>
```